### PR TITLE
Featurebranch

### PR DESCRIPTION
**Pull request by dan626469**
My firs pull request!

### CODE REVIEW

File: text.txt [hunk 1]
```diff
@@ -6,3 +6,5 @@ just commit
 new branch for rebase
 test 2
 the newest commit
```
Comment (danielgerchik) [inline text.txt:8]: Old

File: text.txt [hunk 2]
```diff
@@ -6,3 +6,5 @@ just commit
...
+TEST SSH PUSH
+SPECIAL COMMIT FOR TEST PULL REQUEST
```
Comment (danielgerchik) [inline text.txt:10]: New

Comment (dan626469) [reply text.txt:10]: Okay

**Review by danielgerchik — COMMENTED** [review]
Good work!

**Comment by danielgerchik** [comment]
blabla

**Comment by dan626469** [comment]
lala

**Pull request merged**